apps/passport-server/src/database/models.ts

```diff
@@ -228,7 +229,7 @@ export interface ZuconnectTicketDB {
 }
 
 export interface LinkedPretixTelegramEvent {
-  telegramChatID: string | null;
+  telegramChatID: string | undefined;
```

Comment:
i'm more fine with camelCasing all fields here because `LinkedPretixTelegramEvent` isn't actually a Postgres model, whereas TelegramTopic is

Reply:
Noted and resolved by making a separate type called `FetchTelegramTopic` that is not a 1:1 mapping w the db